app/controllers/user_subscriptions_controller.rb

```diff
@@ -28,6 +28,7 @@ def create
     end
 
     return error_response("subscriber email mismatch") if subscriber_email_stale?
+    return error_response("cannot subscribe with Apple private relay email") if subscriber_authed_with_apple?
```

Comment:
Codeclimate is complaining because I've added too many returns in here. Let's ignore for it now since I already have another issue slated to re-factor this entire controller before release. I had a feeling it would need a few more tweaks over the next couple of PRs so I postponed the refactor. That, and I need to get this all through so the design team can have at it 🎉 .